Applications/Spire/Include/Spire/Ui/NavigationView.hpp

```diff
@@ -0,0 +1,121 @@
+#ifndef SPIRE_NAVIGATION_VIEW_HPP
+#define SPIRE_NAVIGATION_VIEW_HPP
+#include <any>
```

Comment:
Where is ```any``` used?

Reply:
It is used as the parameter type of the function on_list_submit.

Reply:
Good.